pyani_plus/private_cli.py

```diff
@@ -575,5 +575,94 @@ def log_anib(
     return 0
 
 
+@app.command()
+def log_dnadiff(
+    database: Annotated[str, typer.Option(help="Path to pyANI-plus SQLite3 database")],
+    # These are for the comparison table
+    query_fasta: Annotated[Path, typer.Option(help="Path to query FASTA file")],
+    subject_fasta: Annotated[Path, typer.Option(help="Path to subject FASTA file")],
+    mcoords: Annotated[
+        Path, typer.Option(help="Path to show-coords (.mcoords) output file")
+    ],
+    qdiff: Annotated[Path, typer.Option(help="Path to show-diff (.qdiff) output file")],
+    # Should we add --maxmatch (nucmer) and -m (deltafilter) parameters?
+    # These are default parameters used in workflows
+) -> int:
+    """Log a single pyANI-plus ANIm pairwise comparison (with nucmer) to the database."""
+    # Assuming this will match as expect this script to be called right
+    # after the computation has finished (on the same machine)
+    dnadiff_tool = tools.get_dnadiff()
+
+    avg_identity, aligned_bases_with_gaps = method_dnadiff.parse_mcoords(mcoords)
```

Comment:
I'd just use avg_identity -> identity given that's what the database field it is going into is called.